manifests/base/tasks/onboarding.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: task
+metadata:
+  name: run
+spec:
+  params:
+    - name: PAYLOAD
+      type: string
+      default: ""
+  steps:
+    - name: clone-repo
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        echo "Cloning repo"
+        git clone https://github.com/tssala23/ReadIssuesTest.git
+        cd ReadIssuesTest
```

Comment:
This can be part of `apply-push-patch` step.

The way you're cloning the repository here will fail on `git push`. Tekton doesn't know what credentials to use. See this for example how to handle it properly:

https://github.com/operate-first/peribolos-as-a-service/blob/bb1220814f49756fb5eb5b3f2f4b27e219607b1a/manifests/base/tasks/run.yaml#L49